projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -398,12 +395,15 @@ export const enum ScopeQueryParam {
   EndpointTraces = 'endpoint-traces',
   Spans = 'spans'
 }
-const enum ExplorerQueryParam {
+
+export const enum ExplorerQueryParam {
   Scope = 'scope',
   Interval = 'interval',
   Group = 'group',
   OtherGroup = 'other',
   GroupLimit = 'limit',
   Series = 'series',
-  Order = 'order'
+  Order = 'order',
+  // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
Why does the linter think this is shadowing? If it actually is, please adjust name rather than ignoring linter

Reply:
Updated to `Filters`